builtin/credential/okta/backend_test.go

```diff
@@ -41,6 +48,7 @@ func TestBackend_Config(t *testing.T) {
 	username := os.Getenv("OKTA_USERNAME")
 	password := os.Getenv("OKTA_PASSWORD")
 	token := os.Getenv("OKTA_API_TOKEN")
+	createOktaGroups(t, username, token, os.Getenv("OKTA_ORG"))
```

Comment:
Should we clean up these groups?  Looking in Okta I see 201 test groups now.

Reply:
What's the harm in leaving them?

Reply:
No harm, just think it's good to clean up after a test.